configuration/monitor-acs.adoc

```diff
@@ -1,28 +1,37 @@
 [id="monitor-acs"]
-= Monitoring with Prometheus
+= Monitoring {product-title-short}
 include::modules/common-attributes.adoc[]
 :context: monitor-acs
 
 toc::[]
 
 [role="_abstract"]
-link:https://prometheus.io/[Prometheus] is an open-source monitoring and alerting platform.
-You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number `9090` and a network policy allowing inbound connections to that port.
+You can monitor {rh-rhacs-first} by using the built-in monitoring for {osp} or by using custom Prometheus monitoring.
 
-include::modules/enable-monitoring-central-operator.adoc[leveloffset=+1]
+If you use {product-title-short} with {osp}, link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[{ocp}] includes a preconfigured, preinstalled, and self-updating monitoring stack that provides monitoring for core platform components. {product-title-short} exposes metrics to {osp} monitoring via an encrypted and authenticated endpoint.
+
+include::modules/monitoring-osp.adoc[leveloffset=+1]
+include::modules/monitoring-custom-prometheus.adoc[leveloffset=+1]
+include::modules/monitor-osp-disable-operator.adoc[leveloffset=2]
 
 [role="_additional-resources"]
 .Additional resources
 * xref:../installing/installing_ocp/install-central-config-options-ocp.adoc#install-central-config-options-ocp[Central configuration options using the Operator]
 
-include::modules/enable-monitoring-central-helm.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
+include::modules/monitor-osp-disable-helm.adoc[leveloffset=2]
 .Additional resources
 * xref:../installing/installing_other/install-central-other.adoc#change-config-options-after-deployment-central-services_install-central-other[Changing configuration options after deploying the central-services Helm chart]
+* link:https://helm.sh/docs/helm/helm_install/#synopsis[Helm documentation]
 
-include::modules/prometheus-service-monitor.adoc[leveloffset=+1]
+include::modules/enable-monitoring-central-operator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_ocp/install-central-config-options-ocp.adoc#install-central-config-options-ocp[Central configuration options using the Operator]
 
-include::modules/enable-monitoring-manual.adoc[leveloffset=+1]
+include::modules/enable-monitoring-central-helm.adoc[leveloffset=2]
 
-include::modules/enable-monitoring-customize-port.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_other/install-central-other.adoc#change-config-options-after-deployment-central-services_install-central-other[Changing configuration options after deploying the central-services Helm chart]
+* link:https://helm.sh/docs/helm/helm_install/#synopsis[Helm documentation]
```

Comment:
Looks like we have duplicate entries in this sections. How about making a common one with all links and id so we can link from other places?

```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources
* xref:../installing/installing_ocp/install-central-config-options-ocp.adoc#install-central-config-options-ocp[Central configuration options using the Operator]
* xref:../installing/installing_other/install-central-other.adoc#change-config-options-after-deployment-central-services_install-central-other[Changing configuration options after deploying the central-services Helm chart]
* link:https://helm.sh/docs/helm/helm_install/#synopsis[Helm documentation]
```

Reply:
Normally xrefs that go with a particular module are put in the assembly file after the module. Instead, we can do an Additional references for the assembly file as a whole and put them all together. I'll do that and repost and see if it reads better.

Reply:
Here is the guidance about "Additional resources" sections: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections